docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:6.14
+
+## Prepare the image
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update
+RUN apt-get install -y -qq --no-install-recommends unzip
+RUN apt-get clean
```

Comment:
put all the apt-get call into in RUN call: each RUN call results in a separate image so really the cleanup is meaningless on a separate line

same with below and wget, unzip, rm